### PR TITLE
Add v1.3 user stories and use cases to DG

### LINKED ISSUE #111: Add uses cases for enrol/disenrol, filter, mark/unmark, assessment-related features to DG

### LINKED ISSUE #119: Add user stories for enrol/disenrol, filter, mark/unmark, assessment-related features to DG

### LINKED ISSUE #217: Add `duplicates (students) are ignored` for s/ prefix to UG

### PR DESCRIPTION
**Pull request by jxt00**
Closes #119 
Closes #111 
Closes #217 

My bad, the former commit adds user stories while the latter one adds use cases.

The use cases follows the order of the features in the UG.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -615,19 +646,68 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-                <p class="c8"><span class="c0">keep track of students’ attendance</span></p>
+                <p class="c8"><span class="c0">[EPIC] Manage assessments in the modules I am teaching</span></p>
+                <ol class="c10 lst-kix_l5c5afd4kfve-0 start" start="1">
+                    <li class="c7 li-bullet-0"><span class="c0">add assessments to a module</span></li>
+                    <li class="c7 li-bullet-0"><span class="c0">remove assessments from a module</span></li>
+                    <li class="c7 li-bullet-0"><span class="c0">view assessments in a module</span></li>
+                </ol>
+            </td>
+            <td class="c13" colspan="1" rowspan="1">
+                <p class="c8"><span class="c0">easily monitor the assessments I am assigning in a module.</span></p>
```
Comment (wxliong) [inline docs/DeveloperGuide.md:688]: Seems similar to `Manage the assessments I assign` - User story 4?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -660,86 +769,257 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+* 2a. TAssist detects an error in the entered command.
+
+  * 2a1. TAssist prompts for the correct command.
+
+  * 2a2. TA enters a new command.
+
+    Steps 1a1-1a2 are repeated until the command entered is correct.
+
+    Use case resumes from step 2.
```
Comment (wxliong) [inline docs/DeveloperGuide.md:823]: I think this should be step 3 instead

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -660,86 +769,257 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+**Extensions**
+
+* 2a. TAssist detects an error in the entered command.
+
+  * 2a1. TAssist prompts for the correct command.
+
+  * 2a2. TA enters a new command.
+
+    Steps 1a1-1a2 are repeated until the command entered is correct.
```
Comment (wxliong) [inline docs/DeveloperGuide.md:821]: Think there's a typo here, should be 2a1-2a2

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -660,86 +769,257 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+    Use case resumes from step 2.
```
Comment (wxliong) [inline docs/DeveloperGuide.md:847]: Same here!

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -660,86 +769,257 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+**Use case 05: Enrolling student(s)**
+
+**Preconditions:**
+
+1.  The student(s) to be enroled already exist.
+
+2.  The class group that the student(s) will be enroled in already exists.
+
```
Comment (wxliong) [inline docs/DeveloperGuide.md:856]: Minor thing but `enroled` is spelled as `enrolled` in UG, so perhaps can change this to standardize the spelling?

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -660,86 +769,257 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+**Use case 06: Disenrolling student(s)**
 
-      Steps 2b1-2b2 are repeated until the command entered is correct.
+**Preconditions: The student(s) to be disenroled is already enrolled in the class group.**
 
-      Use case resumes from step 3.
+**MSS**
 
-**Use case 04: Add a class group**
+1.  TA requests to disenrol one/more student(s) from a particular class group.
```
Comment (wxliong) [inline docs/DeveloperGuide.md:882]: Same for disenrol 😅

**Review by wxliong — COMMENTED** [review]
Looks good! :) Just some minor things that may need to change

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (jxt00) [reply docs/DeveloperGuide.md:688]: Hmm any suggestions though?

Comment (wxliong) [reply docs/DeveloperGuide.md:688]: Keeping one of them should be enough? May have to check with the others to see if keeping one is sufficient

Comment (jxt00) [reply docs/DeveloperGuide.md:688]: I feel that these are different hence we could keep it. They're related like 1 & 5 & 6, 3 & 7, 4 & 8.

**Review by lkldev — APPROVED** [review]
LGTM

**Review by wxliong — APPROVED** [review]
LGTM! :)

**Pull request merged**